spec/features/catalog_view_spec.rb

```diff
@@ -315,7 +315,12 @@
           expect(page).to have_selector "dd.blacklight-collection_bioghist_ssm", text: /Noël Riley Fitch was born on December 24, 1937/
         end
       end
-
+      it "lets the user search for online materials" do
```

Comment:
This is in the collection page context, but it visits a component page. Could we move it? Maybe near the one I added in #1566?

Reply:
👍 Done.